services/frontend-service/src/ui/components/dropdown/index.ts

```diff
@@ -0,0 +1,19 @@
+/*This file is part of kuberpult.
```

Comment:
This file is unnecessary. Please remove it.

Reply:
Deleted